src/components/Feed/Feed.tsx

```diff
@@ -37,12 +39,32 @@ const Feed = ({ unreadArticles, updateUserSentiment, storeArticle, selectedTopic
     <>
       <Header />
       <div className="articles-container">
+        <button
+          className="arrow-button"
+          disabled={ articleNumber ? false : true }
```

Comment:
Perhaps we can set the arrow to `display: none` when we're at the end of the feed.

Reply:
I like it. Will implement!